asmcli/commands/add-to-mesh.sh

```diff
@@ -72,15 +75,19 @@ EOF
 
 add_all_to_mesh() {
```

Comment:
Is there a way to refactor this into different functions? It's pretty hard to follow.

Reply:
Refactored.